.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,67 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: Provide a general description of the bug, in plain text. Use the dedicated fields below for code snippets.
+      placeholder: Description of the bug
+    validations:
+      required: true
+  - type: textarea
+    id: java-input
+    attributes:
+      label: Source code you are trying to analyze/transform
+      description: Provide a minimal example of source code you are trying to analyze/transform that causes the bug to appear.
+      placeholder: System.out.println("Hello, world!");
+      render: Java
+    validations:
+      required: false
+  - type: textarea
+    id: spoon-code
+    attributes:
+      label: Source code for your Spoon processing
+      description: Provide your processing code that uses Spoon and causes the bug to appear. This will be automatically formatted, so no need for backticks.
+      placeholder: System.out.println("Hello, world!");
+      render: Java
+    validations:
+      required: false
+  - type: textarea
+    id: output
+    attributes:
+      label: Output
+      description: The output that spoon provides, or a stacktrace if it crashes. This will be automatically formatted, so no need for backticks.
+      placeholder: System.out.println("Hello, world!");
+      render: Java
+    validations:
+      required: false
+  - type: textarea
+    id: expected-output
+    attributes:
+      label: Expected output
+      description: If you know what kind of output you expect to get, please provide it here. This will be automatically formatted, so no need for backticks.
+      placeholder: System.out.println("Hello, world!");
+      render: Java
+    validations:
+      required: false
+  - type: textarea
+    id: version
+    attributes:
+      label: Spoon Version
+      description: What version of Spoon are you using? Please note that we only actively support the latest major release. If you are using an older major version, your issue may be closed as non-actionable.
+      placeholder: 10.1.0
```

Comment:
We should add updating this template to the release workflow then.

Reply:
I'm a little bit hesitant here, it's easily forgotten. I _wanted_ to put a dropdown here, but it would be really bad the if we forgot to update it with relevant versions. Even beta versions, which are automatically released.

To cut ourselves loose from a potentially manual step here, perhaps the placeholder should be something like `X.Y.Z` instead?

Reply:
`X.Y.Z` sounds fine, yea. It's probably understandable enough and decidedly less out of date :)